rules/coding-style/tests/Rector/Variable/UnderscoreToCamelCaseLocalVariableNameRector/temp/first_underscore.php.inc

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\Variable\UnderscoreToCamelCaseLocalVariableNameRector\Fixture;
+
+final class FirstUnderscore
+{
+    public function run($_a)
+    {
+        $_some_value = $_a;
+    }
```

Comment:
Could you also add a fixture that tests rename of following use of the variable?

```diff
 public function run($_a)
 {
-    $_some_value = $_a;
+    $someValue = $_a;

-    return $_some_value;
+    return $someValue;
 }
```

Reply:
implemented.

Reply:
I've added `Fixture/param_assign_return.php.inc` for that.